lib/PuppeteerSharp/CDPSession.cs

```diff
@@ -184,22 +184,32 @@ internal void OnMessage(ConnectionResponse obj)
         {
             var id = obj.Id;
 
-            if (id.HasValue && _callbacks.TryRemove(id.Value, out var callback))
+            if (id.HasValue)
```

Comment:
I split this `if` to make this logic match the similar code in Connection.cs. If this is wrong, I can revert.

Reply:
I only suspected that it was an issue, but I haven't observed any issue with the way it was. I'll revert it.